test/AudioParam-test.js

```diff
@@ -491,11 +463,12 @@ describe('AudioParam', function() {
       // t=2 -> t=3 / 0 -> 1
 
       untilTime(audioParam, 2, function(block) {
+        var end = Math.min(128, Math.round((2 - audioParam.context.currentTime) * audioParam.context.sampleRate))
         assertValidBlock(block)
-        assertAllValuesEqual(block.getChannelData(0), -1)
+        assertAllValuesEqual(block.getChannelData(0).slice(0, end), -1)
       })
 
-      var T0 = audioParam.context.currentTime
+      var T0 = 2
```

Comment:
Not sure I follow here

Reply:
I had to change this because `currentTime` progresses in blocks of 128 samples. Which means it's not exactly 2 seconds at this point and that breaks the following computation which assumes the ramp starts exactly at 2 seconds.